lib/rides/models/ride.dart

```diff
@@ -154,6 +159,10 @@ class Ride extends Trip {
     await SupabaseManager.supabaseClient.from('rides').update({'status': status.index}).eq('id', id);
   }
 
+  int getUnreadMessagesCount() {
```

Comment:
not sure where the problem is, but it does not work for me... I think because it considers my own messages as unread?

Reply:
yes exactly that's what i meant in the description

Reply:
But I didn't chat with myself, I just wrote a message to you

Reply:
yes i'm sorry i fixed it. Don't know why i thought i had already done that